internal/configs/version1/nginx.tmpl

```diff
@@ -210,6 +210,17 @@ http {
 
         return 418;
     }
+    {{if .InternalRouteServer}}
+    server {
+        listen 443 ssl;
```

Comment:
```suggestion
        listen 443 ssl;
        {{if not .DisableIPV6}}listen [::]:443 ssl;{{end}}
```

Reply:
@chase-kiefer Should these suggested changes be copied over into the other templates as well? (same for other suggestions)

Reply:
I believe all templates are updated with the suggested changes. I'll have to make sure they come over for the virtualserver work I'm doing though.

Reply:
Do the N+ templates have these? I'm assuming they didn't since you copied this code from those templates.

Reply:
Yes, this matches what n+ has. I must have pulled it from the old MR instead of the newest N+ files. It worked either way but this fits the current pattern used.